_posts/en/2020-11-17-FWD50-in-the-open.md

```diff
@@ -0,0 +1,69 @@
+
+During each session, there was also an opportunity for the users to tag a presenter to ask to network with them later.
+
+Outside the main platform, there were virtual break out rooms where the participants could move their avatar to bump into friends and new acquaintances alike and have the sort of spontaneous conversations that happen at in-person conferences.
+The services used to create those virtual rooms were [Kumospace](https://www.kumospace.com/) and [Wonder](https://www.wonder.me/).
+
+The FWD50 team were very upfront at the beginning to say that there will likely be some hiccups along the way, and while there was one or two minor ones, the conference went very smoothly.
+
+And the good news is that while a paid ticket was required to attend, a lot of content was and still is freely available.
+
+Some of the presenters have made their presentations or details of their presentation open:
+
+- [Pia Andrews](https://twitter.com/piacandrews?lang=en) presented on [The Crisis Paradox For Leadership: Balancing Urgency and Importance](https://t.co/RELqF74GgK?amp=1)
+- [Honey Ducanay's](https://twitter.com/honeygolightly?lang=en) part of the Delivering of the Speed of Need workshop entitled [Enabling conditions, not just heroics](https://t.co/a7u53IwYqs) is available on her blog
+- Also from the 'Delivering at the Speed of Need workshop, panelists [Katherine Benjamin](https://twitter.com/mskatiebenjamin), Honey Ducanay and [Katy Lalonde](https://twitter.com/katylalonde) made their [cheat sheet on delivering at the speed of need](https://docs.google.com/presentation/d/1gL57LdKFFZOm5ngL1HqHubzzG80aLewDcJY8hTDqsaQ/edit#slide=id.p) openly available
+- Simon Wardley's FWD50 presentation isn't public but a lot of [his presentations](https://wardley-maps-community.github.io/awesome-wardley-maps/) are openly available as is his book [Wardley maps: The use of topographical intelligence in business strategy](https://medium.com/wardleymaps/on-being-lost-2ef5f05eb1ec). He also shared a link to a random [strategy generator](http://strategy-madlibs.herokuapp.com/) (the code for which is on GitHub code) which made a lot of us on the IT Strategy team chuckle as each refresh gives a new buzz-wordy nonsense strategy.
+- Ann-Marie Cavanagh, Deputy Government Chief Digital Office and Deputy Chief Executive Digital Public Service, New Zealand, shared her governments [Digital Strategy](https://www.digital.govt.nz/digital-government/strategy/strategy-summary/strategy-for-a-digital-public-service/) and [Digital Inclusion Outcomes Framework](https://www.digital.govt.nz/dmsdocument/167~digital-inclusion-outcomes-framework/html) during her talk on COVID-19 and Economic Recovery: New Zealand's lessons learnt on digital inclusion and resilience.
```

Comment:
Recommend linking "Ann-Marie Cavanagh" to her FWD50 page: https://fwd50.com/speaker/ann-marie-cavanagh/
or to her twitter account: https://twitter.com/amqcs